eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -213,13 +220,20 @@ jobs:
         displayName: Disk Usage after Build
 
     # Build CoreCLR Managed Components
-    - ${{ if or(ne(parameters.osGroup, 'Linux'), ne(parameters.archType, 'x86')) }}:
+    - ${{ if and(ne(parameters.testGroup, 'toolsLinker'), or(ne(parameters.osGroup, 'Linux'), ne(parameters.archType, 'x86'))) }}:
       - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -subset clr.corelib+clr.nativecorelib+clr.nativeaotlibs+clr.tools+clr.packages+clr.paltestlist $(crossArg) -arch $(archType) $(osArg) -c $(buildConfig) $(pgoInstrumentArg) $(officialBuildIdArg) -ci
         displayName: Build managed product components and packages
     - ${{ if and(eq(parameters.osGroup, 'Linux'), eq(parameters.archType, 'x86')) }}:
       - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -subset clr.corelib $(crossArg) -arch $(archType) $(osArg) -c $(buildConfig) $(pgoInstrumentArg) $(officialBuildIdArg) -ci
         displayName: Build managed product components and packages
 
+    # Run Linker unit tests
```

Comment:
Why is it not in some tools yml outside coreclr ?

Reply:
End up using a new runtime-tools-tests.yml for running the tools.linkertests subset